engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/HashCodeNode.java

```diff
@@ -40,21 +51,28 @@
  * <h3>Hashing contract:</h3>
  *
  * <ul>
- *   <li>Whenever two objects are equal ({@code EqualsAnyNode} returns {@code true}), their hashcode
+ *   <li>Whenever two objects are equal ({@code EqualsNode} returns {@code true}), their hashcode
```

Comment:
The rename of `EqualsAnyNode` to `EqualsNode` makes the PR seem bigger than it is and (slightly) complicates the review. Better to do renames as separate PRs.